bfdd/bfd.h

```diff
@@ -449,6 +455,7 @@ void log_init(int foreground, enum blog_level level,
 	      struct frr_daemon_info *fdi);
 void log_info(const char *fmt, ...);
 void log_debug(const char *fmt, ...);
+void log_debug_info(const char *fmt, ...);
```

Comment:
This function name is not very descriptive. How about creating two new debug levels?

1.  `debug bfd control-packet-buffer`: enables/disables debugging messages for buffered control packets;
2.  `debug bfd session-socket`: enables/disables debugging messages for peer sockets (e.g. on all packet `write`/`send`)

Then we'd have:
*   `log_debug_packet_buffer`
*   `log_debug_peer_socket`

Reply:
Certainly creating more debug filters is the way forward.
Currently there is no way to control debugs, for now i thought of creating a switch to control debug, we can take up creation of more debug filters in future.